src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExeWithFSharp.cs

```diff
@@ -27,7 +27,7 @@ public GivenThatWeWantToBuildADesktopExeWithFSharp(ITestOutputHelper log) : base
         {
         }
 
-        [WindowsOnlyFact]
+        [WindowsOnlyFact(Skip = "https://github.com/dotnet/coreclr/issues/27275")]
```

Comment:
I might be able to revert this as they've fixed that bug. I had to ski in other repos, I will send follow up PRs to try to revert it everywhere.